x-pack/platform/plugins/shared/inference/server/chat_complete/anonymization/regex_worker_wrapper.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+const { workerData } = require('worker_threads');
+
+if (process.env.NODE_ENV !== 'production') {
+  /* eslint-disable @kbn/imports/no_boundary_crossing */
+  require('../../../../../../../../src/setup_node_env');
+} else {
+  // eslint-disable-next-line @kbn/imports/no_unresolvable_imports
```

Comment:
I get the `/* eslint-disable @kbn/imports/no_boundary_crossing */`. But I'm not sure if I understand why this is necessary?

Reply:
The paths are different depending on dev or not.  It fails in CI with `Error: Cannot find module '../../../../../../../../src/setup_node_env/dist'` if I use the same paths in dev vs not dev or CI.  When Kibana is built the plugins are in a different directory structure so I adjust for that in the non dev environment.  However this then creates the unresolvable imports issue in linting error.

Reply:
Can you ask operations about this? Surely we have a way to handle this, maybe REPO_ROOT?

Reply:
I had tried REPO_ROOT but it wasn't pointing to the dist as it should have been. I've added it back and it's passing.  Thanks!